great_expectations/validator/validator.py

```diff
@@ -642,17 +640,14 @@ def _build_rule_based_profiler_from_config_and_runtime_args(
 
         override_rules: Optional[
             Dict[str, Dict[str, Any]]
-        ] = override_profiler_config.get("rules")
-        if override_rules is None:
-            override_rules = {}
+        ] = override_profiler_config.get("rules", {})
```

Comment:
Same here please

Reply:
@cdkini Removing `Optional` if that is what you spotted.  Thanks.